src/docfx/build/xref/XrefMap.cs

```diff
@@ -17,7 +17,7 @@ internal class XrefMap
         private readonly IReadOnlyDictionary<string, XrefSpec> _internalXrefMap;
         private readonly IReadOnlyDictionary<string, XrefSpec> _externalXrefMap;
 
-        public IEnumerable<XrefSpec> References => _internalXrefMap?.Values;
+        public IEnumerable<XrefSpec> InternalReferences => _internalXrefMap?.Values;
```

Comment:
other null checks against _internalXrefMap can also be removed

Reply:
yes, they should be removed as well